maven/templates/deploy.py

```diff
@@ -132,18 +139,18 @@ def unpack_args(_, a, b=False):
 
 upload(maven_url, username, password, jar_path, filename_base + '.jar')
 if should_sign:
-    upload(maven_url, username, password, sign(jar_path), filename_base + '.jar.asc')
+    upload(maven_url, username, password, sign(jar_path, gpg_passphrase), filename_base + artifact_extension + '.asc')
```

Comment:
It looks like this change depends on:
- https://github.com/vaticle/bazel-distribution/pull/344

We can either merge this PR by reverting to the following:
```suggestion
    upload(maven_url, username, password, sign(jar_path, gpg_passphrase), filename_base + '.jar.asc')
```
Or wait until https://github.com/vaticle/bazel-distribution/pull/344 is merged.